### ISSUE TITLE
CMake errors when building with the latest camp, umpire, and raja

### ISSUE THREAD

**Issue by adayton1** [LLNL/axom#989]
CMake Error in axom/CMakeLists.txt:
  Target "axom" INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "/g/g17/dayton8/ale3d/update_compiler_branch/imports/axom/src/axom/core/include"

  which is prefixed in the source directory.

CMake Error in axom/CMakeLists.txt:
  Target "axom" INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "/g/g17/dayton8/ale3d/update_compiler_branch/imports/axom/src/axom/lumberjack/include"

  which is prefixed in the source directory.

CMake Error in axom/CMakeLists.txt:
  Target "axom" INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "/g/g17/dayton8/ale3d/update_compiler_branch/imports/axom/src/axom/slic/include"

  which is prefixed in the source directory.

CMake Error in axom/CMakeLists.txt:
  Target "axom" INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "/g/g17/dayton8/ale3d/update_compiler_branch/imports/axom/src/axom/slam/include"

  which is prefixed in the source directory.

CMake Error in axom/CMakeLists.txt:
  Target "axom" INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "/g/g17/dayton8/ale3d/update_compiler_branch/imports/axom/src/axom/primal/include"

  which is prefixed in the source directory.

CMake Error in axom/CMakeLists.txt:
  Target "axom" INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "/g/g17/dayton8/ale3d/update_compiler_branch/imports/axom/src/axom/sidre/include"

  which is prefixed in the source directory.

CMake Error in axom/CMakeLists.txt:
  Target "axom" INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "/g/g17/dayton8/ale3d/update_compiler_branch/imports/axom/src/axom/mint/include"

  which is prefixed in the source directory.

CMake Error in axom/CMakeLists.txt:
  Target "axom" INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "/g/g17/dayton8/ale3d/update_compiler_branch/imports/axom/src/axom/spin/include"

  which is prefixed in the source directory.

CMake Error in axom/CMakeLists.txt:
  Target "axom" INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "/g/g17/dayton8/ale3d/update_compiler_branch/imports/axom/src/axom/inlet/include"

  which is prefixed in the source directory.

CMake Error in axom/CMakeLists.txt:
  Target "axom" INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "/g/g17/dayton8/ale3d/update_compiler_branch/imports/axom/src/axom/klee/include"

  which is prefixed in the source directory.

CMake Error in axom/CMakeLists.txt:
  Target "axom" INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "/g/g17/dayton8/ale3d/update_compiler_branch/imports/axom/src/axom/quest/include"

  which is prefixed in the source directory.

CMake Error in axom/CMakeLists.txt:
  Target "axom" INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "/g/g17/dayton8/ale3d/update_compiler_branch/imports/axom/src/axom/multimat/include"

  which is prefixed in the source directory.

CMake Error in axom/CMakeLists.txt:
  Target "axom" INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "/g/g17/dayton8/ale3d/update_compiler_branch/imports/axom/src/axom/core/include"

  which is prefixed in the source directory.

CMake Error in axom/CMakeLists.txt:
  Target "axom" INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "/g/g17/dayton8/ale3d/update_compiler_branch/imports/axom/src/axom/lumberjack/include"

  which is prefixed in the source directory.

CMake Error in axom/CMakeLists.txt:
  Target "axom" INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "/g/g17/dayton8/ale3d/update_compiler_branch/imports/axom/src/axom/slic/include"

  which is prefixed in the source directory.

CMake Error in axom/CMakeLists.txt:
  Target "axom" INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "/g/g17/dayton8/ale3d/update_compiler_branch/imports/axom/src/axom/slam/include"

  which is prefixed in the source directory.

CMake Error in axom/CMakeLists.txt:
  Target "axom" INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "/g/g17/dayton8/ale3d/update_compiler_branch/imports/axom/src/axom/primal/include"

  which is prefixed in the source directory.

CMake Error in axom/CMakeLists.txt:
  Target "axom" INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "/g/g17/dayton8/ale3d/update_compiler_branch/imports/axom/src/axom/sidre/include"

  which is prefixed in the source directory.

CMake Error in axom/CMakeLists.txt:
  Target "axom" INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "/g/g17/dayton8/ale3d/update_compiler_branch/imports/axom/src/axom/mint/include"

  which is prefixed in the source directory.

CMake Error in axom/CMakeLists.txt:
  Target "axom" INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "/g/g17/dayton8/ale3d/update_compiler_branch/imports/axom/src/axom/spin/include"

  which is prefixed in the source directory.

CMake Error in axom/CMakeLists.txt:
  Target "axom" INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "/g/g17/dayton8/ale3d/update_compiler_branch/imports/axom/src/axom/inlet/include"

  which is prefixed in the source directory.

CMake Error in axom/CMakeLists.txt:
  Target "axom" INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "/g/g17/dayton8/ale3d/update_compiler_branch/imports/axom/src/axom/klee/include"

  which is prefixed in the source directory.

CMake Error in axom/CMakeLists.txt:
  Target "axom" INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "/g/g17/dayton8/ale3d/update_compiler_branch/imports/axom/src/axom/quest/include"

  which is prefixed in the source directory.

CMake Error in axom/CMakeLists.txt:
  Target "axom" INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "/g/g17/dayton8/ale3d/update_compiler_branch/imports/axom/src/axom/multimat/include"

  which is prefixed in the source directory.

CMake Error in axom/CMakeLists.txt:
  Target "axom" INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "/g/g17/dayton8/ale3d/update_compiler_branch/imports/axom/src/axom/core/include"

  which is prefixed in the source directory.

CMake Error in axom/CMakeLists.txt:
  Target "axom" INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "/g/g17/dayton8/ale3d/update_compiler_branch/imports/axom/src/axom/lumberjack/include"

  which is prefixed in the source directory.

CMake Error in axom/CMakeLists.txt:
  Target "axom" INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "/g/g17/dayton8/ale3d/update_compiler_branch/imports/axom/src/axom/slic/include"

  which is prefixed in the source directory.

CMake Error in axom/CMakeLists.txt:
  Target "axom" INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "/g/g17/dayton8/ale3d/update_compiler_branch/imports/axom/src/axom/slam/include"

  which is prefixed in the source directory.

CMake Error in axom/CMakeLists.txt:
  Target "axom" INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "/g/g17/dayton8/ale3d/update_compiler_branch/imports/axom/src/axom/primal/include"

  which is prefixed in the source directory.

CMake Error in axom/CMakeLists.txt:
  Target "axom" INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "/g/g17/dayton8/ale3d/update_compiler_branch/imports/axom/src/axom/sidre/include"

  which is prefixed in the source directory.

CMake Error in axom/CMakeLists.txt:
  Target "axom" INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "/g/g17/dayton8/ale3d/update_compiler_branch/imports/axom/src/axom/mint/include"

  which is prefixed in the source directory.

CMake Error in axom/CMakeLists.txt:
  Target "axom" INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "/g/g17/dayton8/ale3d/update_compiler_branch/imports/axom/src/axom/spin/include"

  which is prefixed in the source directory.

CMake Error in axom/CMakeLists.txt:
  Target "axom" INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "/g/g17/dayton8/ale3d/update_compiler_branch/imports/axom/src/axom/inlet/include"

  which is prefixed in the source directory.

CMake Error in axom/CMakeLists.txt:
  Target "axom" INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "/g/g17/dayton8/ale3d/update_compiler_branch/imports/axom/src/axom/klee/include"

  which is prefixed in the source directory.

CMake Error in axom/CMakeLists.txt:
  Target "axom" INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "/g/g17/dayton8/ale3d/update_compiler_branch/imports/axom/src/axom/quest/include"

  which is prefixed in the source directory.

CMake Error in axom/CMakeLists.txt:
  Target "axom" INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "/g/g17/dayton8/ale3d/update_compiler_branch/imports/axom/src/axom/multimat/include"

  which is prefixed in the source directory.

CMake Error in axom/CMakeLists.txt:
  Target "axom" INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "/g/g17/dayton8/ale3d/update_compiler_branch/imports/axom/src/axom/core/include"

  which is prefixed in the source directory.

CMake Error in axom/CMakeLists.txt:
  Target "axom" INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "/g/g17/dayton8/ale3d/update_compiler_branch/imports/axom/src/axom/lumberjack/include"

  which is prefixed in the source directory.

CMake Error in axom/CMakeLists.txt:
  Target "axom" INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "/g/g17/dayton8/ale3d/update_compiler_branch/imports/axom/src/axom/slic/include"

  which is prefixed in the source directory.

CMake Error in axom/CMakeLists.txt:
  Target "axom" INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "/g/g17/dayton8/ale3d/update_compiler_branch/imports/axom/src/axom/slam/include"

  which is prefixed in the source directory.

CMake Error in axom/CMakeLists.txt:
  Target "axom" INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "/g/g17/dayton8/ale3d/update_compiler_branch/imports/axom/src/axom/primal/include"

  which is prefixed in the source directory.

CMake Error in axom/CMakeLists.txt:
  Target "axom" INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "/g/g17/dayton8/ale3d/update_compiler_branch/imports/axom/src/axom/sidre/include"

  which is prefixed in the source directory.

CMake Error in axom/CMakeLists.txt:
  Target "axom" INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "/g/g17/dayton8/ale3d/update_compiler_branch/imports/axom/src/axom/mint/include"

  which is prefixed in the source directory.

CMake Error in axom/CMakeLists.txt:
  Target "axom" INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "/g/g17/dayton8/ale3d/update_compiler_branch/imports/axom/src/axom/spin/include"

  which is prefixed in the source directory.

CMake Error in axom/CMakeLists.txt:
  Target "axom" INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "/g/g17/dayton8/ale3d/update_compiler_branch/imports/axom/src/axom/inlet/include"

  which is prefixed in the source directory.

CMake Error in axom/CMakeLists.txt:
  Target "axom" INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "/g/g17/dayton8/ale3d/update_compiler_branch/imports/axom/src/axom/klee/include"

  which is prefixed in the source directory.

CMake Error in axom/CMakeLists.txt:
  Target "axom" INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "/g/g17/dayton8/ale3d/update_compiler_branch/imports/axom/src/axom/quest/include"

  which is prefixed in the source directory.

CMake Error in axom/CMakeLists.txt:
  Target "axom" INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "/g/g17/dayton8/ale3d/update_compiler_branch/imports/axom/src/axom/multimat/include"

  which is prefixed in the source directory.

**Comment by white238** [comment]
Spent some time trying to figure out where these are coming from. Gave up and came up with an obviously not ideal, temporary hack:

```
get_target_property(_includes axom INTERFACE_INCLUDE_DIRECTORIES)
foreach(_component ${AXOM_COMPONENTS_ENABLED})
    list(REMOVE_ITEM includes ${PROJECT_SOURCE_DIR}\src\axom\${component}\include}
endforeach()
set_target_properties(axom PROPERTIES INTERFACE_INCLUDE_DIRECTORIES "${includes}")
```

**Comment by adayton1** [comment]
This appears to be an issue coming upstream from Umpire. @white238, should we close this issue here?

**Comment by adayton1** [comment]
Fixed in LLNL/Umpire#805
